### PR TITLE
Update README.md and skype_send_message

### PR DESCRIPTION
**Pull request by idumenskyi**

### CODE REVIEW

File: skype_send_message.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+from skpy import Skype
+import time
+parser = argparse.ArgumentParser()
+parent_parser = argparse.ArgumentParser(add_help=False)
+parent_parser.add_argument('--username', action="store")
+ShDo = "" # variable for complete work, ShDo = "shutdown"
+outputfile = "contact"
+print ("Welcome to 'Skype send message'. ")
+username = str(input("Type your username of Skype: "))
+password = str(input("Type your password of Skype: "))
```
Comment (ekondrashev) [inline skype_send_message.py]: @idumenskyi lets ask for the password if it was not found in the hashed form saved previously

File: skype_send_message.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import bcrypt
+from skpy import Skype
+import time
+parser = argparse.ArgumentParser()
+parent_parser = argparse.ArgumentParser(add_help=False)
+parent_parser.add_argument('--username', action="store")
+ShDo = "" # variable for complete work, ShDo = "shutdown"
+outputfile = "contact"
+print ("Welcome to 'Skype send message'. ")
+username = str(input("Type your username of Skype: "))
```
Comment (ekondrashev) [inline skype_send_message.py]: @idumenskyi `input` usage is not secure, please take a look into `raw_input`
https://stackoverflow.com/questions/4915361/whats-the-difference-between-raw-input-and-input-in-python3-x

File: skype_send_message.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+hashed = bcrypt.hashpw(password.encode('utf8'), bcrypt.gensalt())
+with open(outputfile, mode='wb') as f:
+    f.write(hashed)
+with open(outputfile, mode='rb') as f:
+    f.read()
+contact.user # you
+contact.contacts # your contacts
+contact.chats # your conversations
+message = contact.contacts[str(input("Type user name of Skype for 1-to-1 conversation: "))].chat # 1-to-1 conversation
+while ShDo != 'stop'.upper():
```
Comment (ekondrashev) [inline skype_send_message.py]: @idumenskyi lets simplify this script to one time send only. No need to loop now

File: skype_send_message.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+parent_parser = argparse.ArgumentParser(add_help=False)
+parent_parser.add_argument('--username', action="store")
+ShDo = "" # variable for complete work, ShDo = "shutdown"
+outputfile = "contact"
+print ("Welcome to 'Skype send message'. ")
+username = str(input("Type your username of Skype: "))
+password = str(input("Type your password of Skype: "))
+contact = Skype(username, password)
+hashed = bcrypt.hashpw(password.encode('utf8'), bcrypt.gensalt())
+with open(outputfile, mode='wb') as f:
```
Comment (ekondrashev) [inline skype_send_message.py]: @idumenskyi please consider using `keyring` to store username/password

File: skype_send_message.py (hunk 1 above)
Comment (idumenskyi) [reply skype_send_message.py]: @ekondrashev hashed.find(salt)?

**Pull request merged**